gradio/app.py

```diff
@@ -0,0 +1,94 @@
+import gradio as gr
+from jerboa.utils.prompter import Prompter
+from transformers import (
+    AutoModelForCausalLM,
+    AutoTokenizer,
+    BitsAndBytesConfig,
+    GenerationConfig,
+    AutoConfig,
+)
+from peft import PeftConfig, PeftModel
+import torch
+
+
+# Load model
+QUANT_CONFIG = BitsAndBytesConfig(
+    load_in_8bit=True,
+)
+
+BASE_MODEL = 'tiiuae/falcon-40b'
+LORA_WEIGHTS = '../artifacts/falcon40b_code_alpaca'
+MODEL_CONFIG = AutoConfig.from_pretrained(BASE_MODEL, trust_remote_code=True)
+PEFT_CONFIG = PeftConfig.from_pretrained(
+    LORA_WEIGHTS,
+    trust_remote=True,
+)
+model = AutoModelForCausalLM.from_pretrained(
+    pretrained_model_name_or_path=BASE_MODEL,
+    torch_dtype=torch.float16,
+    device_map='auto',
+    config=MODEL_CONFIG,
```

Comment:
I would expect that this code is the same as the one used in the run code eval, is there a way we can refactor to only have this code once?

Reply:
Ideally, we would even have a small cli to select which model to run

Reply:
Both done